src/System.Net.Http.WinHttpHandler/tests/System.Net.Http.WinHttpHandler.Tests.csproj

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Since you have a buildable src project, can you think the tests against that project instead of including the source?